cmd/bundle/init_test.go

```diff
@@ -25,3 +25,18 @@ func TestBundleInitRepoName(t *testing.T) {
 	assert.Equal(t, "invalid-url", repoName("invalid-url"))
 	assert.Equal(t, "www.github.com", repoName("https://www.github.com"))
 }
+
+func TestNativeTemplateOptions(t *testing.T) {
+	assert.Equal(t, []string{"default-python", "mlops-stacks"}, nativeTemplateOptions())
+}
+
+func TestNativeTemplateDescriptions(t *testing.T) {
+	assert.Equal(t, "- default-python: The default Python template\n- mlops-stacks: The Databricks MLOps Stacks template (https://github.com/databricks/mlops-stacks)", nativeTemplateDescriptions())
```

Comment:
the order is not actually guaranteed. If we want it in ABC order we should sort it. If not, the test should separately assert for containing of each of description